spring-cloud-generator/WORKSPACE

```diff
@@ -0,0 +1,72 @@
+workspace(name = "spring_cloud_generator")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+
+#googleapis
+
+_googleapis_commit = "7438480b2a1bc6371d748e974f7a3647f90c4e8d"
+
+http_archive(
+    name = "com_google_googleapis",
+    strip_prefix = "googleapis-%s" % _googleapis_commit,
+    urls = [
+        "https://github.com/googleapis/googleapis/archive/%s.zip" % _googleapis_commit,
+    ],
+)
+
+# Maven install
+
+RULES_JVM_EXTERNAL_TAG = "4.5"
+RULES_JVM_EXTERNAL_SHA = "b17d7388feb9bfa7f2fa09031b32707df529f26c91ab9e5d909eb1676badd9a6"
+
+http_archive(
+    name = "rules_jvm_external",
+    strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_TAG,
+    sha256 = RULES_JVM_EXTERNAL_SHA,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
+)
+
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
+
+rules_jvm_external_deps()
+
+load("@rules_jvm_external//:setup.bzl", "rules_jvm_external_setup")
+
+rules_jvm_external_setup()
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+
+# Spring generator
+_spring_cloud_generator_version = "4.1.4-SNAPSHOT" # {x-version-update:spring-cloud-gcp:current}
+
+maven_install(
+    artifacts = [
+        "com.google.api:spring-cloud-generator:" + _spring_cloud_generator_version,
+    ],
+    fail_on_missing_checksum = False,
+    repositories = [
+        "m2Local",
+        "https://repo.maven.apache.org/maven2/",
```

Comment:
We can remove this line if we are always going to install it from local

Reply:
Removed in [d9bb09f](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686/commits/d9bb09f1bc7eb9edd5e35ce0ed61028b866ac54e).